pages/accountLists/[accountListId]/tools/useToolsHelper.ts

```diff
@@ -0,0 +1,23 @@
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+import { useAccountListId } from 'src/hooks/useAccountListId';
+import { getQueryParam } from 'src/utils/queryParam';
+
+export const useToolsHelper = () => {
+  const { query, push } = useRouter();
+  const accountListId = useAccountListId();
+  const selectedContactId = getQueryParam(query, 'contactId');
+
+  const handleSelectContact = useCallback(
+    (pagePath: string, contactId: string) => {
+      push(`/accountLists/${accountListId}/${pagePath}/${contactId}`);
```

Comment:
If I load FixSendNewsletter (or any tools page) and then change my account list (say from Pedro to John), when I click on a contact it changes my url to the previous accountListId. So the page refreshes and the contact is not found. I get a snackbar error saying `Contact with id=d44fc289-a918-4319-8b97-9c70ca13c37e not found`